docs/guide/custom_policy.rst

```diff
@@ -128,8 +128,21 @@ Initially shared then diverging: ``[128, dict(vf=[256], pi=[16])]``
            |                |
         action            value
 
+The ``LstmPolicy`` can be used to construct recurrent policies in a similar way:
 
-If however, your task requires a more granular control over the policy architecture, you can redefine the policy directly:
+.. code-block:: python
+
+    class CustomLSTMPolicy(LstmPolicy):
+        def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, n_lstm=64, reuse=False, **_kwargs):
+            super().__init__(sess, ob_space, ac_space, n_env, n_steps, n_batch, n_lstm, reuse,
+                             net_arch=[8, 'lstm', dict(vf=[5, 10], pi=[10])],
+                             layer_norm=True, feature_extraction="mlp", **_kwargs)
+
+Here the ``net_arch`` parameter takes an additional 'lstm' entry within the shared network section.
```

Comment:
Maybe also explain how is the lstm used, it is shared for the two, no?
And is it mandatory? If so, we need to say it.

In your example, if I understand it, you have a shared mlp of size 8 that is follows by a shared lstm of size 64 and then you have different mlp for vf and policy networks?